tests/unit/components/heatmap/HeatMap.spec.js

```diff
@@ -79,15 +79,15 @@ describe("HeatMap.vue", () => {
       localVue,
     });
     const init_heatmap_values = {
-      "Twitch Force": {
+      "Twitch Frequency": {
         data: [
-          new Array(MIN_NUM_DATAPOINTS_FOR_MEAN).fill(1),
-          new Array(MIN_NUM_DATAPOINTS_FOR_MEAN).fill(2),
-          new Array(MIN_NUM_DATAPOINTS_FOR_MEAN).fill(4),
-          new Array(MIN_NUM_DATAPOINTS_FOR_MEAN).fill(5),
+          new Array(MIN_NUM_DATAPOINTS_FOR_MEAN).fill(10),
```

Comment:
I think we should just flip the order of these top level keys

Reply:
We need to keep these values different to make sure the mean is created correctly

Reply:
Please add back different values into each array so the taking the mean is tested. If every value is the same you could rip out taking the mean and grab a random value from these arrays and this test wouldn't fail